### PR TITLE
trt-1204: option to save job run identifier json

### PR DESCRIPTION
**Pull request by neisw**
Adds option to save JobRunIdentifier JSON for further job analysis

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -55,7 +63,7 @@ var cmd = &cobra.Command{
 		for _, envVar := range opts.envVariables {
 			splitVar := strings.SplitN(envVar, "=", 2)
 			if len(splitVar) != 2 {
-				cmd.Usage() //nolint
+				cmd.Usage() // nolint
```
Comment (stbenjam) [inline cmd/main.go:66]: The lack of space is intentional https://github.com/golangci/golangci-lint/issues/3109#issuecomment-1218842579

Comment (stbenjam) [reply cmd/main.go:66]: Although lint seems to pass with the space, not sure why, but directive format in go is // followed by the directive directly without whitespace.

Comment (neisw) [reply cmd/main.go:66]: Yeah, the change wasn't intentional on my part.  But goland is pretty intentional about it...

Comment (stbenjam) [reply cmd/main.go:66]: Weird, golang doesn't touch it for me.

Comment (stbenjam) [reply cmd/main.go:66]: Are you using a go locally less than 1.19?

Comment (neisw) [reply cmd/main.go:66]: go version go1.20 linux/amd64

Comment (neisw) [reply cmd/main.go:66]: I can turn Reformat on save off

Comment (stbenjam) [reply cmd/main.go:66]: I swear lint was failing with the space when I changed it, but if it's passing now we can fix it again if it becomes a problem

**Pull request merged**